src/utils/removeObjectKey.js

```diff
@@ -4,6 +4,6 @@ export default function removeObjectKey ( generator, node, key ) {
 	const index = properties.findIndex( property => property.key.type === 'Identifier' && property.key.name === key );
 	if ( index === -1 ) return;
 	const a = properties[ index ].start;
-	const b = index < properties.length - 1 ? properties[ index + 1 ].start : properties[ index ].end;
+	const b = index < properties.length - 1 ? properties[ index + 1 ].start : node.end - 1;
```

Comment:
Yep, I can't think of a situation where that wouldn't be the case. (I like using https://astexplorer.net to test out AST-related hypotheses, in case you hadn't come across it — really useful)